<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PipeEditorDialog</class>
 <widget class="QDialog" name="PipeEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>393</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>278</width>
    <height>393</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>278</width>
    <height>393</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="mainBox">
     <property name="title">
      <string>Pipe Paramaters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="dimGroupBox">
        <property name="title">
         <string>Dimensions</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QComboBox" name="pipeDimsSelection"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="materialGroup">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>60</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Pipe Material</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QComboBox" name="materialSelection">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="knownsGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Unknowns</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Lenght</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="lengthBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="decimals">
              <number>4</number>
             </property>
             <property name="minimum">
              <double>0.001000000000000</double>
             </property>
             <property name="maximum">
              <double>9999.998999999999796</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>   m</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QCheckBox" name="flowrateCheckbox">
             <property name="text">
              <string>Flow Rate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="flowrateBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-9999.998999999999796</double>
             </property>
             <property name="maximum">
              <double>9999.998999999999796</double>
             </property>
             <property name="value">
              <double>0.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string> L/s</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QCheckBox" name="velocityCheckbox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Velocity</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="velocityBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-9999.998999999999796</double>
             </property>
             <property name="maximum">
              <double>9999.998999999999796</double>
             </property>
             <property name="value">
              <double>0.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>m/s</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QCheckBox" name="pressureInletCheckbox">
             <property name="text">
              <string>Pressure Red</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="pressureInletBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-9999.998999999999796</double>
             </property>
             <property name="maximum">
              <double>9999.998999999999796</double>
             </property>
             <property name="value">
              <double>0.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>kPa</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QCheckBox" name="pressureOutletCheckbox">
             <property name="text">
              <string>Pressure Green</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="pressureOutletBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-9999.998999999999796</double>
             </property>
             <property name="maximum">
              <double>9999.998999999999796</double>
             </property>
             <property name="value">
              <double>0.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>kPa</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
        <zorder></zorder>
        <zorder></zorder>
        <zorder></zorder>
        <zorder></zorder>
        <zorder></zorder>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PipeEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PipeEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>flowrateCheckbox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PipeEditorDialog</receiver>
   <slot>toggleFlowRate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velocityCheckbox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PipeEditorDialog</receiver>
   <slot>toggleVelocity()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pressureInletCheckbox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PipeEditorDialog</receiver>
   <slot>togglePInlet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pressureOutletCheckbox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PipeEditorDialog</receiver>
   <slot>togglePOutlet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>toggleFlowRate()</slot>
  <slot>toggleVelocity()</slot>
  <slot>togglePInlet()</slot>
  <slot>togglePOutlet()</slot>
 </slots>
</ui>
